package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "meanjs",
   "description": "Full-Stack JavaScript with MongoDB, Express, AngularJS, and Node.js.",
-  "version": "0.4.2",
-  "meanjs-version": "0.4.2",
+  "version": "0.4.3",
+  "meanjs-version": "0.4.3",
```

Comment:
@ilanbiala is it required with this change that we increment the meanjs version? or can this version increment wait?